modules/webgl/src/classes/program.js

```diff
@@ -147,10 +147,14 @@ export default class Program extends Resource {
 
     this.gl.useProgram(this.handle);
 
-    // Note: async textures set as uniforms might still be loading.
-    // Now that all uniforms have been updated, check if any texture
-    // in the uniforms is not yet initialized, then we don't draw
-    if (!this._areTexturesRenderable()) {
+    if (
+      // Note: async textures set as uniforms might still be loading.
+      // Now that all uniforms have been updated, check if any texture
+      // in the uniforms is not yet initialized, then we don't draw
+      !this._areTexturesRenderable() ||
+      // Avoid WebGL warning when vertexCount is 0
+      vertexCount <= 0
```

Comment:
Make this `=== 0`. A negative value should be considered invalid, and we'd want that error message to appear. Also, might as well add `instanceCount === 0` just to avoid the useless API call.